src/components/cylc/Drawer.vue

```diff
@@ -99,19 +101,84 @@ export default {
         graphql: mdiGraphql
       },
       environment: process.env.VUE_APP_SERVICES === 'offline' ? 'OFFLINE' : process.env.NODE_ENV.toUpperCase(),
-      version
+      version,
+      navigation: {
+        width: 260,
+        borderSize: 3
+      }
     }
   },
   computed: {
     ...mapState('user', ['user']),
     drawer: {
-      get: function () {
+      get () {
         return this.$store.state.app.drawer
       },
-      set: function (val) {
+      set (val) {
+        if (val) {
+          const newWidth = typeof this.navigation.width === 'string' ? Number(this.navigation.width.replace('px', '')) : this.navigation.width
+          this.navigation.width = newWidth < 260 ? 260 : newWidth
+        }
         this.$store.commit('app/setDrawer', val)
       }
     }
+  },
+  methods: {
+    getDrawerElement () {
+      return this.$refs.drawerRef.$el
+    },
+    setBorderWidth () {
+      const i = this.getDrawerElement().querySelector(
+        '.v-navigation-drawer__border'
+      )
+      i.style.width = this.navigation.borderSize + 'px'
+      i.style.cursor = 'ew-resize'
+    },
+    resize (e) {
+      document.body.style.cursor = 'ew-resize'
+      const el = this.getDrawerElement()
+      const direction = el.classList.contains('v-navigation-drawer--right')
+        ? 'right'
+        : 'left'
+      const f = direction === 'right' ? document.body.scrollWidth - e.clientX : e.clientX
+      el.style.width = f + 'px'
+    },
+    setEvents () {
+      const minSize = this.navigation.borderSize
+      const el = this.getDrawerElement()
+      const drawerBorder = el.querySelector('.v-navigation-drawer__border')
+      drawerBorder.addEventListener(
+        'mousedown',
+        (e) => {
+          if (e.offsetX < minSize) {
```

Comment:
I have figured out this line is what is causing the cypress test to not work properly. The cypress-triggered event is an `Event` rather than `MouseEvent` for a real mouse-down, and so is missing the `offsetX` property.

What is the reason for this `if` statement?

Reply:
This is a really good catch, thank you. I did actually look into the events but didn't make the connection with offsetX. I cant honestly remember why the offsetX was added, but with testing it doesn't seem to be being used so I'll remove it.